File: LightSwitch in Visual Studio 2012 Vision Clinic Walkthrough & Sample/[C#]-LightSwitch in Visual Studio 2012 Vision Clinic Walkthrough & Sample/C#/Vision Clinic CSharp/Common/UserCode/InvoiceDetail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.LightSwitch;
namespace LightSwitchApplication
{
    public partial class InvoiceDetail
    {
        partial void SubTotal_Compute(ref decimal result)
        {
            result = Quantity * UnitPrice;
        }
    }
}

File: LightSwitch in Visual Studio 2012 Vision Clinic Walkthrough & Sample/[VB]-LightSwitch in Visual Studio 2012 Vision Clinic Walkthrough & Sample/VB/Vision Clinic/Common/UserCode/InvoiceDetail.vb

Namespace LightSwitchApplication

    Public Class InvoiceDetail

        Private Sub Product_Changed()
            UnitPrice = Product.CurrentPrice
            Quantity = 1
        End Sub

        Private Sub SubTotal_Compute(ByRef result As Decimal)
            ' Set result to the desired field value
            result = Quantity * UnitPrice
        End Sub
    End Class

End Namespace
